components/ChallengeMaterial.tsx

```diff
@@ -19,43 +24,63 @@ type ChallengeQuestionCardProps = {
 }
 
 type ChallengeMaterialProps = {
-  challenges?: Challenge[]
+  userSubmissions: UserSubmission[]
+  challenges: Challenge[]
 }
 
-export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
-  const { currentState: state, active } = props
-  const cardStyles: string[] = []
+type StatusIconProps = {
+  status: string
+}
+
+const StatusIcon: React.FC<StatusIconProps> = ({ status }) => {
+  if (status === 'unsubmitted') {
+    return <></>
+  }
+  let statusIconUrl
+  switch (status) {
+    case 'passed':
+      statusIconUrl = '/curriculumAssets/icons/checkmark.svg'
+      break
+    case 'needMoreWork':
+      statusIconUrl = '/curriculumAssets/icons/rejected.svg'
+      break
+    case 'open':
+      statusIconUrl = '/curriculumAssets/icons/pending.svg'
+  }
+  return (
+    <>
+      <img width="25px" height="25px" src={statusIconUrl} />
+    </>
+  )
+}
```

Comment:
You don't need to wrap your `<img />` around fragment.
`return <img width="25px" height="25px" src={statusIconUrl} />`